x-pack/plugins/apm/public/components/shared/PropertiesTable/NestedKeyValueTable.tsx

```diff
@@ -45,7 +46,7 @@ const Cell = styled.td`
   }
 
   &:first-child {
-    width: 300px;
+    width: ${units.unit * 20}px;
```

Comment:
I don't have a preference, just thought I should mention we also have a "shorthand" (which is not really shorter):
```
${px(units.unit * 20)}
```

Use whichever you prefer.